lib/utils.js

```diff
@@ -392,6 +392,16 @@ var Utils = module.exports = {
 
   validateParameter: function(value, expectation, options) {
     return ParameterValidator.check(value, expectation, options);
+  },
+
+  formatReferences: function (obj) {
+    if (!_.isPlainObject(obj.references)) {
+      deprecate('Non-object references property. Expected { references: { model: "value", key: "key" } } instead of { references: "value", referencesKey: "key" }.');
+      obj.references = { model: obj.references, key: obj.referencesKey };
+      delete obj.referencesKey;
```

Comment:
Please just set this to undefined. There are different ways how objects are handled depending on what you do with them and if you delete a object property it'll automatic enter the slow dictionary mode. The operation itself takes a lot longer too as you can see here: http://jsperf.com/delete-vs-undefined-vs-null/45